src/Components/blazor/samples/MonoSanity/MonoSanity.csproj

```diff
@@ -6,7 +6,6 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore" Version="$(MicrosoftAspNetCoreAppPackageVersion)" />
-    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="$(MicrosoftAspNetCoreRazorDesignPackageVersion)" />
```

Comment:
This package is obsolete in 3.0 apps and isn't built anymore. Only 2.0 apps should reference it. As discussed though, it's still included by `.App`. This will go away when we do a shared framework.